examples/distance-matrix/components/distance-matrix/distance-matrix.tsx

```diff
@@ -0,0 +1,116 @@
+import React, {useEffect, useState} from 'react';
+
+import {
+  useDistanceMatrix,
+  useGeocoder,
+  useGoogleMap
+} from '@ubilabs/google-maps-react-hooks';
+
+import {getOriginMarkerIcon} from '../../libs/marker-icon-helpers';
+
+import styles from './distance-matrix.module.css';
+
+const DistanceMatrix = () => {
+  const {map} = useGoogleMap();
+  const geocoder = useGeocoder();
+
+  // Get the service from the useDistanceMatrix hook
+  const service = useDistanceMatrix();
+
+  const [elements, setElements] = useState<
+    google.maps.DistanceMatrixResponseElement[]
+  >([]);
+  const [destinationList, setDestinationList] = useState(['']);
+
+  useEffect(() => {
+    if (!map || !service) {
+      return;
+    }
+
+    const bounds = new google.maps.LatLngBounds();
+
+    // Distance Matrix request
+    // https://developers.google.com/maps/documentation/distance-matrix/distance-matrix#required-parameters
+    const origin = 'Invalides';
+    const destination1 = 'Palais Garnier';
+    const destination2 = {lat: 48.87486838960709, lng: 2.293475122446004}; // Arc de Triomphe
+    const destination3 = {lat: 48.85680070130583, lng: 2.2917585087353705}; // Eiffel Tower
+    const destination4 = 'Louvre Museum';
+    const destination5 = {lat: 48.846182892343855, lng: 2.3604230571607023}; // Jardin de Plantes
+
+    const request = {
+      origins: [origin],
+      destinations: [
+        destination1,
+        destination2,
+        destination3,
+        destination4,
+        destination5
+      ],
```

Comment:
```suggestion
    const destinations = [
      'Palais Garnier',
      {lat: 48.87486838960709, lng: 2.293475122446004}, // Arc de Triomphe
      {lat: 48.85680070130583, lng: 2.2917585087353705}, // Eiffel Tower
      'Louvre Museum',
      {lat: 48.846182892343855, lng: 2.3604230571607023} // Jardin de Plantes
    ];

    const request = {
      origins: [origin],
      destinations,
```

... or do you see any advantages in using separate variables?

Reply:
I personally think it has a better readability. But I get that it might be a bit redundant :/